src/foam/nanos/theme/Theme.js

```diff
@@ -100,6 +100,23 @@ foam.CLASS({
       name: 'appName',
       section: 'infoSection',
     },
+    {
+      class: 'String',
+      name: 'loginDAO'
+    },
+    {
+      class: 'String',
+      name: 'loginImage'
+    },
+    {
+      class: 'String',
+      name: 'defaultGroup'
+    },
+    {
+      class: 'Array',
+      of: 'String',
+      name: 'loginCountryChoices'
```

Comment:
Drop this. We'll have Crunch and CountryCorridors working before the next deployment.

Reply:
How would I drop this without breaking the SignUp? Wouldn't it be better to keep it as is and then refactor when Crunch/CountryCorridors is good to go?

Reply:
on foam, don't restrict.  On nanopay refine or extend and hard code.  It will be fixed by 'application' wide country corridor settings. if we add it to Theme it will never be removed.

Reply:
Got it!